src/overlays/actors/ovl_En_Test3/z_en_test3.c

```diff
@@ -1214,33 +1214,56 @@ void EnTest3_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList1, Gfx** dL
     }
 }
 
-static TexturePtr sEyeTextures[] = {
-    gKafeiEyesOpenTex,     gKafeiEyesHalfTex,   gKafeiEyesClosedTex,   gKafeiEyesRollRightTex,
-    gKafeiEyesRollLeftTex, gKafeiEyesRollUpTex, gKafeiEyesRollDownTex, object_test3_Tex_006680,
+static TexturePtr sEyeTextures[PLAYER_EYES_MAX] = {
+    gKafeiEyesOpenTex,    // PLAYER_EYES_OPEN
+    gKafeiEyesHalfTex,    // PLAYER_EYES_HALF
+    gKafeiEyesClosedTex,  // PLAYER_EYES_CLOSED
+    gKafeiEyesRightTex,   // PLAYER_EYES_RIGHT
+    gKafeiEyesLeftTex,    // PLAYER_EYES_LEFT
+    gKafeiEyesUpTex,      // PLAYER_EYES_UP
+    gKafeiEyesDownTex,    // PLAYER_EYES_DOWN
+    gKafeiEyesWincingTex, // PLAYER_EYES_WINCING
 };
 
-static TexturePtr sMouthTextures[] = {
-    gKafeiMouthClosedTex,
-    gKafeiMouthTeethTex,
-    gKafeiMouthAngryTex,
-    gKafeiMouthHappyTex,
+static TexturePtr sMouthTextures[PLAYER_MOUTH_MAX] = {
+    gKafeiMouthClosedTex, // PLAYER_MOUTH_CLOSED
+    gKafeiMouthHalfTex,   // PLAYER_MOUTH_HALF
+    gKafeiMouthOpenTex,   // PLAYER_MOUTH_OPEN
+    gKafeiMouthSmileTex,  // PLAYER_MOUTH_SMILE
 };
 
-typedef struct {
-    /* 0x0 */ u8 eyeIndex;
-    /* 0x1 */ u8 mouthIndex;
-} KafeiFace; // size = 0x2
-
-static KafeiFace sFaceExpressions[] = {
-    { 0, 0 }, { 1, 0 }, { 2, 0 }, { 0, 0 }, { 1, 0 }, { 2, 0 }, { 4, 0 }, { 5, 1 }, { 7, 2 }, { 0, 2 },
-    { 3, 0 }, { 4, 0 }, { 2, 2 }, { 1, 1 }, { 0, 2 }, { 0, 0 }, { 0, 0 }, { 0, 0 }, { 0, 0 }, { 0, 0 },
+static PlayerFaceIndices sKafeiFaces[PLAYER_FACE_MAX] = {
+    // The first 6 faces defined must be default blinking faces. See relevant code in `Player_UpdateCommon`.
+    { PLAYER_EYES_OPEN, PLAYER_MOUTH_CLOSED },   // PLAYER_FACE_NEUTRAL
+    { PLAYER_EYES_HALF, PLAYER_MOUTH_CLOSED },   // PLAYER_FACE_NEUTRAL_BLINKING_HALF
+    { PLAYER_EYES_CLOSED, PLAYER_MOUTH_CLOSED }, // PLAYER_FACE_NEUTRAL_BLINKING_CLOSED
```

Comment:
I was unsure whether to reuse the player face enums/structs for Kafei or to make a near identical copy for kafei. There is nothing specific tying these together.

Reply:
I think I'm inclined to just reuse the enums/structs. Don't really have any motivation other than I don't really see why we need to duplicate them.